Oleksii_Makarov/HW8/Lesson8_4.py

```diff
@@ -0,0 +1,22 @@
+def my_max(item: iter) -> iter:
+    """
+    Reverse sorts a list and returns first position
+    """
+    maxi = sorted(item, reverse=True)
+    return maxi[0]
```

Comment:
with this approach, it is easy to implement * task as well. Try to think about it